google_shopping.rb

```diff
@@ -7,9 +7,45 @@
 #snake case word_word not camel case wordWord
 google_data = JSON.parse(file)
 
+data = google_data['items']
+backorder = []
+images = []
+count = 0
+canon_product = []
```

Comment:
I'm going to be really nitpicky here and suggest using the plural, ```canon_products``` as well as ```ebay_products``` to show people reading your code that it's a collection.